src/BIMDataComponents/BIMDataDropdownMenu/BIMDataDropdownMenu.vue

```diff
@@ -22,9 +22,18 @@
               v-for="item in menuItems"
               :key="item.name"
               class="bimdata-dropdown__elements__menu-items__item"
-              @click="item.action && item.action()"
-              @mouseover="handleCurrentItem(item.name)"
-              @mouseleave="handleCurrentItem()"
+              :style="{
+                color: hasNoChildren(item.children)
+                  ? 'var(--color-silver-dark)'
+                  : 'var(--color-primary)',
+              }"
+              @click.stop="
+                !hasNoChildren(item.children) && item.action && item.action()
+              "
+              @mouseover="
+                !hasNoChildren(item.children) && handleCurrentItem(item.name)
+              "
+              @mouseleave="!hasNoChildren(item.children) && handleCurrentItem()"
```

Comment:
3 x `!hasNoChildren` for 1 `hasNoChildren` , `hasChildren`may be more appropriate.